src/main/java/com/example/customerdb/controller/PurchaseInfoController.java

```diff
@@ -16,16 +15,33 @@ public class PurchaseInfoController {
     public PurchaseInfoController(PurchaseInfoService purchaseInfoService) {
         this.purchaseInfoService = purchaseInfoService;
     }
-    @GetMapping("/purchase-info")
+    @GetMapping("/")
```

Comment:
購入情報を表していると思っていたのですが、だとしたらこちらの方が良いかと思いました。
何を扱うAPIなのかパスから判断できるのがよいですね。
```suggestion
    @GetMapping("/purchase-info")
```